core/domain/user_services_test.py

```diff
@@ -132,6 +137,52 @@ def test_get_user_id_from_username(self):
         self.assertIsNone(
             user_services.get_user_id_from_username('fakeUsername'))
 
+    def test_fetch_gravatar_success(self):
+        user_email = 'user@example.com'
+        gravatar_filepath = os.path.join(
```

Comment:
expected_gravatar_filepath (otherwise it becomes a bit hard to remember what's observed and what's expected when reading the test)